stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel_actions/ChannelActionsDialogFragment.kt

```diff
@@ -0,0 +1,146 @@
+package io.getstream.chat.android.ui.channel_actions
+
+import android.content.Context
+import android.os.Bundle
+import android.text.format.DateUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.isVisible
+import androidx.fragment.app.viewModels
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamFragmentChannelActionsBinding
+
+internal class ChannelActionsDialogFragment : BottomSheetDialogFragment() {
+    private val membersAdapter: ChannelMembersAdapter = ChannelMembersAdapter {
+        listener?.onMemberSelected(it)
+    }
+
+    private val cid: String by lazy { requireArguments().getString(ARG_CID)!! }
+    private val isGroup: Boolean by lazy { requireArguments().getBoolean(ARG_IS_GROUP, false) }
+
+    private var listener: OnChannelActionSelectedListener? = null
+
+    private val channelActionsViewModel: ChannelActionsViewModel by viewModels {
+        ChannelActionsViewModelFactory(cid, isGroup)
+    }
+
+    private var _binding: StreamFragmentChannelActionsBinding? = null
```

Comment:
We're not clearing this `_binding` variable anywhere

Reply:
Nice catch

Reply:
![](https://media.giphy.com/media/3og0ILv9Mvflt6tgxG/giphy.gif)